packages/compiler-ssr/src/ssrCodegenTransform.ts

```diff
@@ -60,6 +60,8 @@ export function ssrCodegenTransform(ast: RootNode, options: CompilerOptions) {
     ...ast.helpers.filter(h => h in ssrHelpers),
     ...context.helpers
   ]
+  // make unique after merge
+  ast.ssrHelpers = Array.from(new Set(ast.ssrHelpers))
```

Comment:
We can merge them into one

```suggestion
  ast.ssrHelpers = Array.from(new Set([
    ...ast.helpers.filter(h => h in ssrHelpers),
    ...context.helpers
  ]))
```

Reply:
merged and removed the comment about unique, I hope the code is self describing here.